source/_integrations/sonos.markdown

```diff
@@ -302,6 +302,8 @@ action:
 
 To work optimally, the Sonos devices must be able to connect back to the Home Assistant host on TCP port 1400. This will allow the push-based updates to work properly. If this port is blocked or otherwise unreachable from the Sonos devices, the integration will fall back to a polling mode which is slower to update and much less efficient. The integration will alert the user if this problem is detected.
 
+As of 2023.5, playing audio using the `announce` option or TTS requires TCP port 1443 on each Sonos device to be reachable from the Home Assistant host.
```

Comment:
We don't keep a changelog in the documentation. Our documentation reflects the current stable version of Home Assistant.

```suggestion
Playing audio using the `announce` option or TTS requires TCP port 1443 on each Sonos device to be reachable from the Home Assistant host.
```